R/PartInv.R

```diff
@@ -94,180 +113,174 @@ NULL
-#'                 labels = c("female", "male"),
-#'                 show_mi_result = TRUE)
+#' PartInv(propsel = .10,
+#'         weights_item = c(1/3, 1/3, 1/3, 1/3),
+#'         weights_latent = 1,
+#'         alpha_r = 0.5,
+#'         alpha_f = 0,
+#'         psi_r = 1,
+#'         lambda_r = c(.3, .5, .9, .7),
+#'         nu_r = c(.225, .025, .010, .240),
+#'         nu_f = c(.225, -.05, .240, -.025),
+#'         Theta_r = diag(.96, 4),
+#'         labels = c("female", "male"),
+#'         show_mi_result = TRUE)
+#' # Multiple groups, multiple dimensions
+#' lambda_matrix <- lambda_matrix1 <- lambda_matrix2 <- matrix(0, nrow = 5, ncol = 2)
+#' lambda_matrix[1:2, 1] <-  c(.322, .655); lambda_matrix[3:5, 2] <- c(.398, .745, .543)
+#' lambda_matrix1[1:2, 1] <-  c(.392, .665); lambda_matrix1[3:5, 2] <- c(.388, .725, .523)
+#' lambda_matrix2[1:2, 1] <-  c(.372, .650); lambda_matrix2[3:5, 2] <- c(.368, .7, .543)
+#' # 2 dimensions, 5 items (2,3), 3 groups
+#' PartInv(propsel = 0.25, cut_z = 2, pmix = c(1/3, 1/3, 1/3),
+#'        alpha = list(c(1,1), c(0,0), c(.5, .5)),
+#'        psi = list(c(1, 0.2, 0.2, 1), c(1, 0.3, 0.3, 1), c(1, 0.4, 0.4, 1)),
+#'        nu = list(c(rep(1,5)), c(rep(1.5,5)),c(rep(1.2,5))),
+#'        lambda = list(lambda_matrix, lambda_matrix1, lambda_matrix2),
+#'        Theta = list(c(rep(.1,5)), c(rep(.4,5)), c(rep(.3,5))),
+#'        plot_contour = TRUE, labels = c("Group 1", "Group 2", "Group 3"),
+#'        custom_colors = c("salmon1", "lightgreen", "skyblue1"), 
+#'        show_mi_results = TRUE)
 #' @export
 PartInvMulti_we <- function(propsel = NULL, cut_z = NULL,
                             weights_item = NULL,
                             weights_latent = NULL,
+                            alpha, psi, lambda, Theta, nu,
+                            pmix = 0.5,
+                            pmix_ref = 0.5, plot_contour = FALSE,
+                            show_mi_result = FALSE,
+                            labels = NULL,
                             kappa_r = NULL, kappa_f = kappa_r,
-                            alpha_r, alpha_f = alpha_r,
+                            alpha_r = NULL, alpha_f = alpha_r,
                             phi_r = NULL, phi_f = phi_r,
-                            psi_r, psi_f = psi_r,
-                            lambda_r, lambda_f = lambda_r,
+                            psi_r = NULL, psi_f = psi_r,
+                            lambda_r = NULL, lambda_f = lambda_r,
                             tau_r = NULL, tau_f = tau_r,
-                            nu_r, nu_f = nu_r,
-                            Theta_r, Theta_f = Theta_r,
-                            pmix_ref = 0.5, plot_contour = FALSE,
-                            show_mi_result = FALSE,
-                            labels = c("Reference", "Focal"), ...) {
+                            nu_r = NULL, nu_f = nu_r,
+                            Theta_r = NULL, Theta_f = Theta_r,
+                            ...) {
```

Comment:
It seems that the `show_mi_results` parameter was lost here, and it results in a warning when plotting on my end when running the example

Reply:
Warnings are generated in the plot